src/screens/Settings/NotificationSettings/ActivityNotificationSettings.tsx

```diff
@@ -0,0 +1,127 @@
+
+type Props = NativeStackScreenProps<
+  AllNavigatorParams,
+  'ActivityNotificationSettings'
+>
+export function ActivityNotificationSettingsScreen({}: Props) {
+  const t = useTheme()
+  const {_} = useLingui()
+  const {data: preferences, isError} = useNotificationSettingsQuery()
+
+  // TODO: Fetch subscriptions
+  const items = [] as string[]
+
+  const renderItem = useCallback(() => {
+    return null
+  }, [])
+
+  return (
+    <Layout.Screen>
+      <Layout.Header.Outer>
+        <Layout.Header.BackButton />
+        <Layout.Header.Content>
+          <Layout.Header.TitleText>
+            <Trans>Notifications</Trans>
+          </Layout.Header.TitleText>
+        </Layout.Header.Content>
+        <Layout.Header.Slot />
+      </Layout.Header.Outer>
+      <List
+        ListHeaderComponent={
+          <SettingsList.Container>
+            <SettingsList.Item style={[a.align_start]}>
+              <SettingsList.ItemIcon icon={BellRingingIcon} />
+              <ItemTextWithSubtitle
+                bold
+                titleText={<Trans>Activity alerts</Trans>}
+                subtitleText={
+                  <Trans>
+                    Activity Alerts notify you instantly when your favorite
+                    accounts post or reply.
+                  </Trans>
+                }
+              />
+            </SettingsList.Item>
+            {isError ? (
+              <View style={[a.px_lg, a.pt_md]}>
+                <Admonition.Admonition type="error">
+                  <Trans>Failed to load notification settings.</Trans>
+                </Admonition.Admonition>
+              </View>
+            ) : (
+              <PreferenceControls
+                name="subscribedPost"
+                preference={preferences?.subscribedPost}
+              />
+            )}
+          </SettingsList.Container>
+        }
+        data={items}
+        renderItem={renderItem}
+        ListEmptyComponent={
+          <View style={[a.px_xl, a.py_md]}>
+            <Admonition.Outer type="tip">
+              <Admonition.Row>
+                <Admonition.Icon />
+                <View style={[a.flex_1, a.gap_sm]}>
+                  <Admonition.Text>
+                    <Trans>
+                      Enable activity alerts for an account by visiting their
+                      profile and pressing the{' '}
+                      <Text style={[a.font_bold, t.atoms.text_contrast_high]}>
+                        bell icon
+                      </Text>{' '}
+                      <BellIcon size="xs" style={t.atoms.text_contrast_high} />.
+                    </Trans>
+                  </Admonition.Text>
+                  <Admonition.Text>
+                    <Trans>
+                      By default, only accounts you follow can receive alerts
+                      from you – this can be changed in{' '}
+                      <InlineLinkText
+                        label={_(msg`Privacy & Security settings`)}
```

Comment:
```suggestion
                        label={_(msg`Privacy and Security settings`)}
```
nit: Even though I'm guessing it doesn't make a difference when read out by a screen reader, it might make sense to simply use `and` here too